NitroxPatcher/Patches/Dynamic/LaunchRocket_OnHandClick_Patch.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxClient.GameLogic;
+using NitroxClient.Unity.Helper;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public class LaunchRocket_OnHandClick_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((LaunchRocket t) => t.OnHandClick(default));
+
+    public static bool Prefix(LaunchRocket __instance)
+    {
+        // Copied code from the method itself
```

Comment:
Would use a transpiler here. It should be very simple since you can trigger at `LaunchRocket.SetLaunchStarted()` and then call a static method and return.